src/main/java/com/uipath/uipathpackage/UiPathPack.java

```diff
@@ -64,6 +70,12 @@ public UiPathPack(SelectEntry version, String projectJsonPath, String outputPath
         this.outputPath = outputPath;
         this.traceLevel = traceLevel;
         this.outputType = "None";
+        this.splitOutput = false;
```

Comment:
I remember we decided to set the dafault value of `splitOutput` to `false` in order to avoid a breaking change.
But I don't remember what was that breaking change. Do you remember?

Reply:
The default value for `splitOutput` to false was to convey backwards compatibility for clients that used the Deploy task with a specific `*.nupkg` parameter and not the directory.

Reply:
Oh, ok. So the CLI supports getting a nupkg absolute path instead of a directory.
Thanks.